docs/specification.md

```diff
@@ -365,7 +365,15 @@ As an open system the ISCC allows any person or organization to offer ISCC regis
 
 ### Blockchain Registry
 
-A well known, decentralized, open, and public registry for canonical discoverability of ISCC identified content is of great value. For this reason it is RECOMMENDED to register ISCC identifiers on the open `iscc` data-stream of the [Content Blockchain](https://content-blockchain.org/). For details please refer to the [ISCC-Stream specification](https://coblo.github.io/cips/cip-0003-iscc/) of the Content Blockchain.
+To properly address the questions of identifier uniqueness, ownership and authentication within the ISCC Standard, the assignment of a canonical blockchain is a requirement. The distributed nature of blockchains are a perfect fit for long-term persistent identifier registration and resolver services.
+
+**The assignment of a canonical blockchain is NOT YET part of this specification.**
+
+Because this decision is of such vital importance, we would like to wait for more community involvement before these questions are addressed either in an updated version of this specification or in a separate specification. 
```

Comment:
"Because this decision is of such vital importance, we suggest to wait for further feedback and additional community involvement before these questions are addressed either in an updated version of this specification or in a separate specification.